domains/nass.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "SpauriRosso",
+           "email": "spy456@zoho.com",
+           "discord": "404659098919895045"
+        },
+    
+        "record": {
+            "A": ["64.226.79.220"]
```

Comment:
Can't access site, please give a preview of your website.

Reply:
Hello after verifying website is accessible at http://64.226.79.220/home
From this project https://github.com/SpauriRosso/GroupieTracker
![image](https://github.com/user-attachments/assets/c16295bb-8da5-4f1e-8359-b3c7d93ff773)